src/main/java/seedu/duke/account/Account.java

```diff
@@ -0,0 +1,297 @@
+                                AccountHistoryFile.updateLoginAccount(wallet.getUserName(), loginTime, in);
+                            }
+                            else{
+                                throw new FinanceException(ExceptionCollection.COMMAND_TYPE_EXCEPTION);
+                            }
+                    }
+
+                } catch (FinanceException e) {
+                    e.handleException();
+                }
+            }
+            else if (splits.length == 3) {
+                String commandType = splits[0];
+                String commandArg = splits[1] + " " + splits[2];
+                try {
+                    switch (commandType) {
+
+                        case "save":
+                            MoneyCommand.saveCommand(wallet,commandArg);
+                            AccountHistoryFile.updateLoginAccount(wallet.getUserName(), loginTime, in);
+                            break;
+                        case "withdraw":
+                            MoneyCommand.withdrawCommand(wallet, commandArg);
+                            AccountHistoryFile.updateLoginAccount(wallet.getUserName(), loginTime, in);
+                            break;
+                        default:
+                            throw new FinanceException(ExceptionCollection.COMMAND_TYPE_EXCEPTION);
+                    }
+
+                } catch (FinanceException e) {
+                    e.handleException();
+                }
+            }
+            else if (splits.length == 4){
+                String commandType = splits[0];
+                String recipientUsername = splits[1];
+                int amount = Integer.parseInt(splits[2]);
+                String transferCurrency = splits[3];
+
+                try {
+
+                    switch (commandType) {
+                    case "convert":
+                        MoneyCommand.exchangeCommand(wallet, splits[1]+" "+splits[2]+" "+splits[3]);
+                        AccountHistoryFile.updateLoginAccount(wallet.getUserName(), loginTime, in);
+                        break;
+                    case "transfer":
+                        MoneyCommand.transferMoney(wallet, recipientUsername, transferCurrency, amount);
+                        AccountUi.showTransfer(recipientUsername, transferCurrency, amount);
+                        AccountHistoryFile.updateLoginAccount(wallet.getUserName(), loginTime, in);
+                        break;
+                    default:
+                        throw new FinanceException(ExceptionCollection.COMMAND_TYPE_EXCEPTION);
+                    }
+                } catch (FinanceException e) {
+                    e.handleException();
+                }
+            }
+            else {
+                throw new FinanceException(ExceptionCollection.COMMAND_TYPE_EXCEPTION);
+            }
+        }
+        return isAccountExit;
+    }
+
+    private boolean helpCenter(){
```

Comment:
Again, this method is too long. It could be simplified into several sub-routines.